.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 tmp
+local.json
+social-media
```

Comment:
Why add `social-media` in gitignore when we are needing it in tests?

Reply:
In the original code, the `tmp` folder which had the screenshots of the sign up page was included in gitignore, and since the social media folder serves the same function, I thought the same convention was supposed to be followed with it.

Reply:
I don't think it being in gitignore is of any use as such 🤔 
The folder is already included in your branch 🤔